sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/ActivityExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using OpenTelemetry.Exporter.AzureMonitor.Models;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    internal static class ActivityExtensions
+    {
+        private static readonly string INVALID_SPAN_ID = default(ActivitySpanId).ToHexString();
+        private static readonly string INVALID_TRACE_ID = default(ActivityTraceId).ToHexString();
+
+        internal static string GetSpanId(this ActivitySpanId activitySpanId)
+        {
+            var spanId = activitySpanId.ToHexString();
+            if (!string.Equals(spanId, INVALID_SPAN_ID, StringComparison.Ordinal))
+            {
+                return spanId;
+            }
+
+            return null;
+        }
+
+        internal static string GetTraceId(this ActivityTraceId activityTraceId)
+        {
+            var traceId = activityTraceId.ToHexString();
+            if (!string.Equals(traceId, INVALID_TRACE_ID, StringComparison.Ordinal))
+            {
+                return traceId;
+            }
+
+            return null;
+        }
+
+        internal static TelemetryType GetTelemetryType(this Activity activity)
+        {
+            if (activity.Kind == ActivityKind.Server || activity.Kind == ActivityKind.Consumer)
```

Comment:
switch/case seems like a better fit, especially if extended for other types (for readability). You also have the benefits of a jump table from switch.

Reply:
There is a TODO work, pending in this method. Will change this to switch in next PR.